snapshots/energy_institute/2023-06-26/statistical_review_of_world_energy.csv.dvc

```diff
@@ -0,0 +1,23 @@
+meta:
+  name: Energy Institute Statistical Review of World Energy 2023
+  publication_year: 2023
+
+  publication_date: '2023-06-26'
+  source_name: Energy Institute Statistical Review of World Energy 2023
+  source_published_by: Energy Institute Statistical Review of World Energy 2023
+  url: https://www.energyinst.org/statistical-review/
+  source_data_url: 
+    https://www.energyinst.org/__data/assets/file/0007/1055761/Consolidated-Dataset-Panel-format-CSV.csv
+  license_url: 
+    https://www.energyinst.org/__data/assets/pdf_file/0004/1055542/EI_Stat_Review_PDF_single-2.pdf
```

Comment:
This link to the license isn't working for me, getting a 404.

Reply:
Ah it looks like they may have already changed the link to: https://www.energyinst.org/__data/assets/pdf_file/0004/1055542/EI_Stat_Review_PDF_single_3.pdf

Perhaps a good idea to use a web archive link in case they do this again.

Reply:
Thanks for pointing this out. That's a bit annoying, but the only place I've found where they mention citation and copyrights is the PDF. I've just changed it to the link from the web archive.